src/hooks/chart.ts

```diff
@@ -346,6 +392,10 @@ const parsePlotData = (
       );
       additionalLayout = layout;
       return parseResult;
+    } else if (plot.plot_type === 'heatmap') {
+      const heatmapData = parseHeatmapResponse(currTrace, plot, responseData, additionalLayout);
```

Comment:
No need to pass `additionalLayout` into this function since it's an empty object.